modules/github-webhook/main.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  enabled = module.this.enabled
+
+  fetch_github_webhook = local.enabled && !(length(var.webhook_github_secret) > 0)
+
+  # If fetching webhook, get the value from SSM
+  # Else, get the value given by var.webhook_github_secret
+  webhook_github_secret = local.fetch_github_webhook ? try(data.aws_ssm_parameter.webhook[0].value, null) : var.webhook_github_secret
+}
+
+data "aws_ssm_parameter" "webhook" {
```

Comment:
Use `remote-state` pattern to pass the webhook value

Reply:
We shouldnt store secrets in terraform output

Reply:
I guess we can use `sensitive`

Reply:
added sensitive output and updated `github-webhook` to use either ssm or remote-state. remote-state by default